### PR TITLE
Refactors E2E test which verifies ARO attributes

### PR DESCRIPTION
**Pull request by m1kola**
### Which issue this PR addresses:

Part of [WI 13667437](https://msazure.visualstudio.com/AzureRedHatOpenShift/_workitems/edit/13667437).

### What this PR does / why we need it:

Refactors E2E test which verifies ARO attributes

### Test plan for issue:

E2E run

### Is there any documentation that needs to be updated for this PR?

No, just E2E refactoring.

### CODE REVIEW

File: test/e2e/aro.go [hunk 1]
```diff
@@ -18,62 +18,59 @@ import (
...
+		for _, mc := range mcs.Items {
+			actualMachineConfigNames = append(actualMachineConfigNames, mc.Name)
 		}
-		By("Checking if workerSsh and masterSsh is enabled or disabled")
-		Expect(workerSsh).To(BeTrue())
-		Expect(masterSsh).To(BeTrue())
+		Expect(actualMachineConfigNames).To(ContainElements([]string{
+			"90-aro-worker-registries",
+			"99-master-aro-dns",
+			"99-master-fips",
```
Comment (bennerv) [inline test/e2e/aro.go:40]: `fips` isn't guaranteed, it's keyed off of the cluster document -> `properties.clusterProfile.fipsValidatedModules == "Enabled"`

Can we put in a check to include the `99-master|worker-fips` machineconfigs iff the property is enabled in the clusterProfile?

Comment (bennerv) [reply test/e2e/aro.go:40]: Technically we're creating all E2E clusters currently with FIPS enabled, but if we change that, someone will come back and discover this.  :smile:

**Review by bennerv — COMMENTED** [review]
one nit - otherwise lgtm

**Comment by m1kola** [comment]
@bennerv updated. Now the condition is in place. Please take another look.

**Review by bennerv — APPROVED** [review]
Lgtm

**Pull request merged**